torch/jit/frontend.py

```diff
@@ -363,6 +378,94 @@ def build_param(ctx, py_arg, self_name, kwarg_only):
         annotation_expr = EmptyTypeAnnotation(r)
     return Param(annotation_expr, Ident(r, name), kwarg_only)
 
+def build_ignore_context_manager(ctx, stmt):
+    def process_ins_outs(args):
+        # parse the context manager to figure out inputs and outputs
+        # with their annotated types
+        inputs = []
+        outputs = []
+        for arg in args:
+            var_name = arg.arg
+            var_ann = arg.value.value
+            var_decl_type, var_ann = var_ann.split(":")
+            if var_decl_type == "inp":
+                inputs.append((var_name, var_ann))
+            if var_decl_type == "out":
+                outputs.append((var_name, var_ann))
+        return inputs, outputs
+
+    def create_unique_name_ext(ctx, stmt):
+        # extension will be based on the filename plus the line number
+        # of original context manager
+        return os.path.basename(ctx.filename).replace(".", "_") + "_" + str(stmt.lineno)
+
+    def build_return_ann_stmt(outputs):
+        return_type_ann = ""
+        return_statement_str = "return "
+        if len(outputs) == 0:
+            return_type_ann += " -> None: pass"
```

Comment:
Could you refactor the code a bit to make `pass` not part of `return_type_ann`, which is supposed to be just annotation

Reply:
yep